apps/dashboard/src/routes/lms/community/ask/+page.svelte

```diff
@@ -66,8 +87,25 @@
     </div>
   </div>
 
-  <div class="mb-3 p-2">
-    <TextField bind:value={fields.title} placeholder="Title" errorMessage={errors.title} />
+  <div class="mb-3 p-2 flex gap-x-5 justify-between">
+    <TextField
+      bind:value={fields.title}
+      placeholder="Title"
+      errorMessage={errors.title}
+      className="w-[75%]"
+    />
+    <Dropdown
+      class="w-[25%]"
+      size="xl"
+      label="Select Course"
+      invalid={isInvalid}
+      items={fetchedCourses.map((course) => ({ id: course.id, text: course.title }))}
+      bind:selectedId={fields.course_id}
+      on:select={(event) => {
+        const { selectedId } = event.detail;
+        fields.course_id = selectedId;
```

Comment:
@tunny17 What is this logic for?

Reply:
i've deleted it, it's gone.

Reply:
Yes I know why you call the fetchCourses function, however you don't use `coursesResults` anywhere in the code which is why I can't understand why the variable is there

Reply:
the courseResults only stores the data from fetchCourse and fetchedCourses gets the allCourses property from the courseResults object

Reply:
I see that, but for you to create a top variable outside the function scope, it has to be used to render the UI or used in another function. I asking to make you realize that it isn't needed.

Please change it to:

```js
async function getCourses(userId: string | null, orgId: string) {
  const fetchCoursesResponse = await fetchCourses(userId, orgId);
  fetchedCourses = fetchCoursesResponse.allCourses;
}
```

Then remove `coursesResults` from the file and also do that in any other place this implemented

Reply:
oh, okay, noted